<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventManager</class>
 <widget class="QWidget" name="EventManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="1" column="0">
    <widget class="QGroupBox" name="gb_Event">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>212</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QStackedWidget" name="sw_Event">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page_Empty"/>
        <widget class="QWidget" name="page_Profil">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QPushButton" name="btn_DebProfil">
            <property name="text">
             <string>Début de profil</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" rowspan="3">
           <widget class="QStackedWidget" name="sw_profil">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="page_Empty"/>
            <widget class="QWidget" name="page_Edit">
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="4">
               <widget class="QSpinBox" name="sp_NumProfil">
                <property name="maximum">
                 <number>99999</number>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Fichier</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLineEdit" name="le_fichier"/>
              </item>
              <item row="0" column="3">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>N° de profil</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2" colspan="3">
               <widget class="QPlainTextEdit" name="te_CommentsProfil"/>
              </item>
              <item row="1" column="2">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Commentaires</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="btn_FinProfil">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Fin de profil</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_Sippican">
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0">
           <widget class="QStackedWidget" name="sw_Sippican">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="page_Empty">
             <layout class="QGridLayout" name="gridLayout_11">
              <item row="0" column="0">
               <widget class="QPushButton" name="btn_Sippican">
                <property name="text">
                 <string>Tir Sippican</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <spacer name="horizontalSpacer_6">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>370</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="0">
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>103</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_Edit">
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>N° sonde Sippican</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSpinBox" name="sp_nbSippican"/>
              </item>
              <item row="0" column="2" colspan="2">
               <widget class="Line" name="line_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
               </widget>
              </item>
              <item row="0" column="4" colspan="2">
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>Type de sonde:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="8">
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Fichier</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1" colspan="7">
               <widget class="QLineEdit" name="le_FicSippican">
                <property name="minimumSize">
                 <size>
                  <width>291</width>
                  <height>20</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Température SBE (°C)</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QDoubleSpinBox" name="sp_SBE">
                <property name="decimals">
                 <number>1</number>
                </property>
                <property name="value">
                 <double>15.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="2" column="2" rowspan="2">
               <widget class="Line" name="line_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
               </widget>
              </item>
              <item row="2" column="3" colspan="3">
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>Sonde Navire (m)</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="6">
               <widget class="QSpinBox" name="sp_SondeNav">
                <property name="maximum">
                 <number>9999</number>
                </property>
                <property name="value">
                 <number>10</number>
                </property>
               </widget>
              </item>
              <item row="2" column="7" colspan="2">
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>134</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Célérimètre (m/s)</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QDoubleSpinBox" name="sp_Celerimetre">
                <property name="decimals">
                 <number>1</number>
                </property>
                <property name="minimum">
                 <double>1000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>2000.000000000000000</double>
                </property>
                <property name="value">
                 <double>1500.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="3" column="3" colspan="3">
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>Célérité ASVP à 3m (m/s)</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="3" column="6">
               <widget class="QDoubleSpinBox" name="sp_CelASVP">
                <property name="decimals">
                 <number>1</number>
                </property>
                <property name="minimum">
                 <double>1000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>2000.000000000000000</double>
                </property>
                <property name="value">
                 <double>1500.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QPushButton" name="btn_LoadSip">
                <property name="text">
                 <string>Chargement EM2040</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="4" column="1" colspan="3">
               <widget class="QDateTimeEdit" name="dte_LoadSip"/>
              </item>
              <item row="4" column="4">
               <widget class="Line" name="line_4">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
               </widget>
              </item>
              <item row="4" column="5">
               <widget class="QLabel" name="label_18">
                <property name="text">
                 <string>Fichier EM2040</string>
                </property>
               </widget>
              </item>
              <item row="4" column="6" colspan="2">
               <widget class="QLineEdit" name="le_Fic2040">
                <property name="minimumSize">
                 <size>
                  <width>113</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="4" column="8">
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>61</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>Commentaires:</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1" colspan="8">
               <widget class="QLineEdit" name="le_CommentSip"/>
              </item>
              <item row="0" column="6">
               <widget class="QLineEdit" name="le_TypeSonde">
                <property name="maximumSize">
                 <size>
                  <width>60</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_Tech">
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QStackedWidget" name="sw_NewEvent">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="page">
             <layout class="QGridLayout" name="gridLayout_13">
              <item row="0" column="1">
               <spacer name="horizontalSpacer_7">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>346</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="0">
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>103</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="0">
               <widget class="QPushButton" name="btn_NewEvent">
                <property name="text">
                 <string>Nouvel évènement</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_2">
             <layout class="QGridLayout" name="gridLayout_12">
              <item row="0" column="0">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Evénement</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="le_Event"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Commentaires:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QPlainTextEdit" name="te_CommentEvt"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_AchiContinu">
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="0" column="2" rowspan="3">
           <widget class="QStackedWidget" name="sw_ArcCont">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="page_3">
             <layout class="QGridLayout" name="gridLayout_15"/>
            </widget>
            <widget class="QWidget" name="page_4">
             <layout class="QGridLayout" name="gridLayout_14">
              <item row="0" column="0">
               <widget class="QLabel" name="label_19">
                <property name="text">
                 <string>Commentaires:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPlainTextEdit" name="te_CommentArcContinu"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item row="2" column="0">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>103</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="btn_startAcqui">
            <property name="text">
             <string>Début Acquisition</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="btn_StopAcqui">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Fin Acquisition</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QStackedWidget" name="sw_DataRetenues">
        <property name="maximumSize">
         <size>
          <width>357</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page_Empty"/>
        <widget class="QWidget" name="page_Edit">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="2" column="0">
           <widget class="QPushButton" name="btn_Valider">
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Valider</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0" colspan="5">
           <widget class="QGroupBox" name="groupBox">
            <property name="maximumSize">
             <size>
              <width>368</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="title">
             <string>Données retenues</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0" colspan="2">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Date et Heure:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3" colspan="2">
              <widget class="QDateTimeEdit" name="sp_dtRetenue">
               <property name="maximumSize">
                <size>
                 <width>159</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Position :</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3" colspan="2">
              <widget class="QLineEdit" name="le_PosRetenue">
               <property name="maximumSize">
                <size>
                 <width>165</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Route fond :</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QSpinBox" name="sp_COGRetenu">
               <property name="maximumSize">
                <size>
                 <width>73</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="maximum">
                <number>360</number>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="Line" name="line">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item row="2" column="3">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Vitesse fond (nd):</string>
               </property>
              </widget>
             </item>
             <item row="2" column="4">
              <widget class="QSpinBox" name="sp_SOGRetenu">
               <property name="maximumSize">
                <size>
                 <width>73</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="3">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>109</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="4">
           <widget class="QPushButton" name="btn_Annuler">
            <property name="text">
             <string>Annuler</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="btn_MajData">
            <property name="text">
             <string>Maj Données</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="l_EventName">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
